playground/infrastructure/cloudbuild/playground_deploy_cloudbuild_part_1.yaml

```diff
@@ -0,0 +1,111 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  - name: ubuntu
+    args:
+      - '-c'
+      - >-
+        apt-get update >/dev/null && apt-get install -y git build-essential
+        unzip software-properties-common ca-certificates curl wget gnupg
+        lsb-release openjdk-11-jdk >/dev/null
+
+        curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
+
+        add-apt-repository "deb [arch=amd64]
+        https://download.docker.com/linux/ubuntu focal stable" > /dev/null 
```

Comment:
why `focal`? Please use the actually used version of the system when adding a repo